.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/awslabs/ecs-logs-collector/blob/master/CONTRIBUTING.md
+
+Please provide the following information:
+-->
+
+### Summary
+<!-- What does this pull request do? -->
+
+### Implementation details
+<!-- How are the changes implemented? -->
+
+### Testing
+<!-- How was this tested? -->
+- [ ] Works properly on Amazon Linux
+- [ ] Works properly on RHEL 7
+- [ ] Works properly on Debian 8
+- [ ] Works properly on Ubuntu 14.04
+
+New tests cover the changes: <!-- yes|no -->
+
+### Description for the changelog
```

Comment:
There isn't a `CHANGELOG` in this repo and this tool isn't versioned, I think this should be left out until that changes.